named_arrays/_matrices/matrices.py

```diff
@@ -33,8 +34,17 @@ def components(self) -> dict[str, na.AbstractVectorArray]:
 
     @property
     def entries(self) -> dict[tuple[str, str], na.ScalarLike]:
```

Comment:
The type signature has changed, now it can return `dict[tuple[str, ...], na.ScalarLike]`